test/XRP/web-socket-network-client.test.ts

```diff
@@ -110,7 +110,7 @@ describe('WebSocket Tests', function (): void {
         address,
         'subscribe_transaction_' + address,
         // eslint-disable-next-line @typescript-eslint/no-empty-function
-        (_data: WebSocketTransactionResponse) => {},
+        (_data: TransactionResponse) => {},
```

Comment:
will the compiler complain if you just remove the param? ie just change this to `() => {}`